#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_gcc4_fix.dpatch by Andreas Jochens <aj@andaco.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: gcc 4 fix

--- nethack-3.4.3/win/Qt/qt_win.cpp~	2005-07-15 13:19:44.000000000 +0300
+++ nethack-3.4.3/win/Qt/qt_win.cpp	2005-07-15 13:19:45.000000000 +0300
@@ -4437,7 +4437,7 @@
 // XXX       define "-" to do "E-yyyyyyyy\r", but would still need "-" for
 // XXX       other purposes.  Maybe just too bad.
 //
-struct {
+static struct {
     int key;
     int state;
     const char* macro;

